SpanishFilter/sections/antiadblock.txt

```diff
@@ -7,6 +7,12 @@
 ! Good: @@/adblocker/detect.js$domain=example.org
 ! Bad: ||ad.doubleclick.net^ (should be in adservers.txt)
 !
+!
+! https://github.com/AdguardTeam/AdguardFilters/issues/134176
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$redirect=googlesyndication-adsbygoogle,domain=canalhelptutoryt.blogspot.com
+!#if (adguard_app_ios || adguard_ext_android_cb || adguard_ext_safari)
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=canalhelptutoryt.blogspot.com
+!#endif
```

Comment:
> Note, you should specify the type of the request for the redirect rule.

Okay.

> Btw, your rules does not work on my end. Try something different like this, for instance

In fact, I tested with AdGuard Base filter[^1], AdGuard Chinese and AdGuard Spanish/Portuguese filter[^2].
[^1]: The anti-adblock script on the website detects the filter on my end. In other words, only AG Spanish + AG Chinese are not detected.
[^2]: I enabled the filter because the document is written in Portuguese language.

Reply:
It does not matter on my end, I still see it here https://canalhelptutoryt.blogspot.com/.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/200821315-5dec133d-044d-4b97-b700-72c6168759f3.png)

</details><br/>